src/main/mdo/dependency-updates-report.mdo

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<model xmlns="http://codehaus-plexus.github.io/MODELLO/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://codehaus-plexus.github.io/MODELLO/2.0.0 https://codehaus-plexus.github.io/modello/xsd/modello-2.0.0.xsd"
+       xsd.namespace="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0"
+       xsd.targetNamespace="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0">
```

Comment:
it is not schema for rules ...

Reply:
Oops! Okay, corrected.

Reply:
What do you think of the following?

http://mojo.codehaus.org/versions-maven-plugin/dependency-updates-report/1.0.0
http://mojo.codehaus.org/versions-maven-plugin/plugin-updates-report/1.0.0